src/mongoc/mongoc-write-concern.c

```diff
@@ -205,6 +205,7 @@ mongoc_write_concern_set_wtimeout (mongoc_write_concern_t *write_concern,
                                    int32_t            wtimeout_msec)
 {
    bson_return_if_fail(write_concern);
+   bson_return_if_fail(wtimeout_msec >= 0);
```

Comment:
This is prone to printing the following on stderr when running the tests with a debug build:

```
mongoc_write_concern_set_wtimeout(): precondition failed: wtimeout_msec >= 0
```

I talked with @ajdavis about the downsides of using `bson_return_if_fail()` to check user input, noting that it is compiled out of non-debug builds; however, we agreed it best to deal with that in a separate ticket (likely an epic).

In this case, is there anything I can do to avoid spamming stderr when tests are executed? Do any other tests trigger these `bson_return_*()` conditions?